docs/api/extras/core/Path.html

```diff
@@ -1,123 +1,125 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
It's dos/unix end of line inconsistency. Same issue here https://github.com/mrdoob/three.js/pull/6859#issuecomment-122063407